pyface/resource/resource_manager.py

```diff
@@ -113,15 +113,12 @@ def _locate_image(self, image_name, resource_path, size):
             # If we come across a reference to a module, use pkg_resources
             # to try and find the image inside of an .egg, .zip, etc.
             if isinstance(dirname, types.ModuleType):
-                from pkg_resources import resource_string
-
+                from importlib_resources import files
```

Comment:
Could this go to the top-level?

Also I think CI is complaining that it cannot import `files` from `importlib_resources`:
```
ImportError: Failed to import test module: pyface.tests.test_image_resource
Traceback (most recent call last):
  File "/home/travis/.edm/envs/pyface-test-3.6-pyqt/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/home/travis/.edm/envs/pyface-test-3.6-pyqt/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/home/travis/.edm/envs/pyface-test-3.6-pyqt/lib/python3.6/site-packages/pyface/tests/test_image_resource.py", line 17, in <module>
    from importlib_resources import files
ImportError: cannot import name 'files'
```

Reply:
yep because we're missing the latest `importlib_resources`

Reply:
If this relies on a specific version of `importlib_resources` then I think this needs to be specified in the requirements. Then we would expect installation from EDM to fail because the required version is not available.